p2p/transport/webrtc/connection.go

```diff
@@ -0,0 +1,275 @@
+package libp2pwebrtc
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"sync"
+
+	ic "github.com/libp2p/go-libp2p/core/crypto"
+	"github.com/libp2p/go-libp2p/core/network"
+	"github.com/libp2p/go-libp2p/core/peer"
+	tpt "github.com/libp2p/go-libp2p/core/transport"
+	ma "github.com/multiformats/go-multiaddr"
+	"github.com/pion/webrtc/v3"
+)
+
+var _ tpt.CapableConn = &connection{}
```

Comment:
I suppose this is a interface test, might as well document that fact

Reply:
This is fine, that's pretty standard way of doing a compile-time interface assertion in Go.

Reply:
I do understand that. But personally I prefer that to be explicitly made clear with a line comment so there is no doubt about this, and put it next to the struct, instead of isolated from one another.

But more of a minor remark, nothing big.